### ISSUE TITLE
Loading `modisom` with only required/needed packages triggers a bunch of warnings

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/modisom#21]
```
Syntax warning: Unbound global variable in /home/runner/gap/pkg/modisom-3.0.0/\
gap/grpalg/tabletoalgebraandback.gi:5
    if Augmentation(el) <> Zero(k) then
       ^^^^^^^^^^^^
Syntax warning: Unbound global variable in /home/runner/gap/pkg/modisom-3.0.0/\
gap/grpalg/tabletoalgebraandback.gi:20
    sup := Support(el);
           ^^^^^^^
Syntax warning: Unbound global variable in /home/runner/gap/pkg/modisom-3.0.0/\
gap/grpalg/tabletoalgebraandback.gi:21
    coeffs := CoefficientsBySupport(el);
              ^^^^^^^^^^^^^^^^^^^^^
```

**Comment by olexandr-konovalov** [comment]
All three are provided by the LAGUNA package.

**Comment by margollo** [comment]
Yes, so probably it is the best solution to load LAGUNA when loading modisom. Is it better to make a new version even for this small change?

**Comment by olexandr-konovalov** [comment]
@margollo that means, make LAGUNA required package. Yes, that warrants a new release IMO.

**Comment by margollo** [comment]
Has been resolved in version 3.0.1
